### ISSUE TITLE
Change inner tag `cit` doesn't work with nested tags

### ISSUE THREAD

**Issue by RileyTomasek** [t9md/atom-vim-mode-plus#646]
Atom version
```
Atom    : 1.12.7
Electron: 1.3.13
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

vim-mode-plus version: 0.66.1

### Problem:
When using `cit` for nested tags on the same line, the inner tag is always changed.

### Example:
```
<span>outer<span>inner</span></span>
```

### Current behavior:
When I type `cit` with the cursor over any character in `outer`, `inner` is removed with the cursor between the inner span `<span>outer<span>{cursor}</span></span>`.

### Expected behavior:
When I type `cit` with the cursor over any character in `outer`, the entire contents of the outer span are cleared and I am left with just `<span>{cursor}</span>`

Please let me know if this isn't clear or if anything else would be helpful to identify what is causing this.

Thanks!

**Comment by t9md** [comment]
Understood your point, thank you for reporting. will check.

**Comment by t9md** [comment]
How I fixed for future record for me.
- [Strategy] If lineTextTillTagStart contains NON white-space
  - We treat it cursor is IN outer-tag.(new)
  - lineTextTillTagStart: text from column 0 of tagStart's row to tagStarting position
- [Implementation]
  - When tagFinder found forwarding range and lineTextTillTagStart contains non-WS
  - Retry to re-find enclosing(outer-tag) tag range rather than forwarding range
  

![tagfinderimprove](https://cloud.githubusercontent.com/assets/155205/21748208/648b0080-d5c2-11e6-9a25-c93495b35c5f.gif)

**Comment by t9md** [comment]
Fixed but new release will be available from only Atom-1.13(currently in-beta).
